### PR TITLE
browser: fix disposition value for new-window event

### LINKED ISSUE #4760: "disposition" should be "background-tab" in "new-window" event when clicking link with ctrl/cmd pressed
* Electron version: v0.36.9
* Operating system: Windows 10/OS X

In #779 a `disposition` was added to the `new-window` event in [webContents](https://github.com/atom/electron/blob/master/docs/api/web-contents.md#event-new-window) but it always seems to be set to `new-window` when clicking on target="_blank" links in our web app wrapper: https://github.com/irccloud/irccloud-desktop/blob/v0.1.3/app/main.js#L109

If I Ctrl (Win) or Cmd (OS X) or middle click on links, I would expect to see a `background-tab` disposition, as in Chrome that will open links in a background tab, while keeping focus on the current tab. I'd like to catch this behaviour to set `activate` to false in `shell.openExternal`.

Is this possible?

### PR DESCRIPTION
**Pull request by deepak1556**
Fixes #4760 

~~Depends on https://github.com/atom/libchromiumcontent/pull/187~~

Depends on https://github.com/atom/libchromiumcontent/pull/191

### CODE REVIEW

File: atom/browser/api/atom_api_app.cc [hunk 1]
```diff
@@ -275,6 +276,53 @@ void App::SelectClientCertificate(
...
+                          bool user_gesture,
+                          bool opener_suppressed,
+                          content::ResourceContext* context,
+                          int render_process_id,
+                          int opener_render_view_id,
+                          int opener_render_frame_id,
+                          bool* no_javascript_access) {
+  DCHECK_CURRENTLY_ON(content::BrowserThread::IO);
+
+  content::BrowserThread::PostTask(content::BrowserThread::UI, FROM_HERE,
```
Comment (zcbenz) [inline atom/browser/api/atom_api_app.cc]: We can put this logic in `AtomBrowserClient::CanCreateWindow`, so we only need to have `App::OnCreateWindow` .

**Comment by zcbenz** [comment]
I have merged the patch in libchromiumcontent, can you also update it here?

**Comment by deepak1556** [comment]
Have made the changes, thanks!

**Comment by zcbenz** [comment]
👍

**Comment by bridiver** [comment]
method sig has been update in chromium to match
https://codereview.chromium.org/2206963002

**Pull request merged**